setup.py

```diff
@@ -1,16 +1,21 @@
 """Project Setup"""
 
 from setuptools import setup
+import config
+
+with open("README.md", 'r') as f:
+    long_description = f.read()
+
+with open('LICENSE') as f:
```

Comment:
Although License is not there, this [PR}(https://github.com/amruthvvkp/aladdin/pull/5) will bring in a license file